src/containers/Section.js

```diff
@@ -111,32 +125,57 @@ const Section = ({ id, mendata, womendata, color, current }) => {
 
   return (
     <div>
-      <Heading />
-      <GenderWrap id='section1'>
-        <Filter onClick={() => setSection("mens")}>Men's Basketball</Filter>
-        <Filter onClick={() => setSection("womens")}>Women's Basketball</Filter>
+      <Heading setSection={setSection} />
```

Comment:
passed through setSection here. The path was heading -> VerticalNav

It wasn't enough just adding a prop to verticalNav if you didnt pass it in yet